osdk/src/util.rs

```diff
@@ -163,6 +163,10 @@ pub fn get_current_crate_info() -> CrateInfo {
     let metadata = get_cargo_metadata(None::<&str>, None::<&[&str]>).unwrap();
 
     let default_member = get_default_member(&metadata);
+    parse_default_member(default_member)
+}
+
+pub fn parse_default_member(default_member: &str) -> CrateInfo {
```

Comment:
It is only used to parse the default member but actually it parses the [Package ID](https://doc.rust-lang.org/cargo/commands/cargo-metadata.html). All the package sources including the members from `cargo metadata` are given in this format. So I suggest renaming the function to `parse_package_id_string`.